src/components/studies/StudyBuilder.tsx

```diff
@@ -138,6 +141,58 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
   const studyDataUpdateFn = useStudyInfoDataDispatch()
   const [open, setOpen] = React.useState(true)
   const [displayBanner, setDisplayBanner] = React.useState(false)
+  const [bannerTypes, setBannerTypes] = React.useState(
+    new Map<
+      string,
+      {
+        bgColor: string
+        displayText: string[]
+        icon: string[]
+        textColor: string
+      }
+    >()
+  )
+
+  React.useEffect(() => {
+    const bannerMap = new Map()
+    bannerMap.set('error', {
+      bgColor: '#EE6070',
+      displayText: ['The following fields are required to launch your study.'],
+      icon: [Alert_Icon],
+      textColor: 'white',
+    })
+    bannerMap.set('success', {
+      bgColor: '#AEDCC9',
+      displayText: ['Page has been saved successfully.'],
+      icon: [SaveIcon],
+      textColor: 'black',
+    })
+    bannerMap.set('live', {
+      bgColor: '#2196F3',
+      displayText: [
+        'This page is view only. We currently don’t support scheduling related editing of lives studies. To make adjustments to your study, create a duplicate of this study and launch again.',
+        'You may edit this page.',
+      ],
+      icon: [NoEditIcon],
+      textColor: 'white',
+    })
+    bannerMap.set('completed', {
+      bgColor: '#EE6352',
+      displayText: ['The study is officially closed and cannot be edited.'],
+      icon: [NoEditIcon],
+      textColor: 'white',
+    })
+    bannerMap.set('withdrawn', {
+      bgColor: '#AA00FF',
+      displayText: ['The study was cancelled and cannot be edited.'],
+      icon: [NoEditIcon],
+      textColor: 'white',
+    })
+    setBannerTypes(bannerMap)
```

Comment:
should the bannerTypes be in the state? are they dynamic? It would make sense to have bannerType in the state instead of /bannerTypes.get(bannerType)' function calls.